ydb/tests/functional/tenants/test_users_groups_with_acl.py

```diff
@@ -0,0 +1,129 @@
+            script_client = ydb.ScriptingClient(driver)
+            script_client.execute_yql('CREATE USER user;')
+            script_client.execute_yql('ALTER USER user WITH PASSWORD NULL;')
+            script_client.execute_yql('DROP USER user;')
+
+    def query_create_user(self, admin_driver_config):
+        with ydb.Driver(admin_driver_config) as driver:
+            with ydb.QuerySessionPool(driver, size=1) as pool:
+                pool.execute_with_retries("CREATE USER user;")
+                pool.execute_with_retries('ALTER USER user WITH PASSWORD NULL;')
+                pool.execute_with_retries("DROP USER user;")
+
+    def yql_create_group(self, admin_driver_config):
+        with ydb.Driver(admin_driver_config) as driver:
+            driver.wait(5)
+            script_client = ydb.ScriptingClient(driver)
+            script_client.execute_yql("CREATE GROUP group;")
+            script_client.execute_yql("CREATE USER user;")
+            script_client.execute_yql("ALTER GROUP group ADD USER user;")
+            script_client.execute_yql("ALTER GROUP group DROP USER user;")
+            script_client.execute_yql("DROP GROUP group;")
+            script_client.execute_yql("DROP USER user;")
+
+    def query_create_group(self, admin_driver_config):
+        with ydb.Driver(admin_driver_config) as driver:
+            with ydb.QuerySessionPool(driver, size=1) as pool:
+                pool.execute_with_retries("CREATE GROUP group;")
+                pool.execute_with_retries("CREATE USER user;")
+                pool.execute_with_retries("ALTER GROUP group ADD USER user;")
+                pool.execute_with_retries("ALTER GROUP group DROP USER user;")
+                pool.execute_with_retries("DROP GROUP group;")
+                pool.execute_with_retries("DROP USER user;")
+
+    def test_yql_create_user_by_domain_admin(self):
+        self.yql_create_user(self.domain_admin_driver_config)
+
+    def test_yql_create_user_by_tenant_admin(self):
+        self.yql_create_user(self.tenant_admin_driver_config)
+
+    def test_yql_create_group_by_domain_admin(self):
+        self.yql_create_group(self.domain_admin_driver_config)
+
+    def test_yql_create_group_by_tenant_admin(self):
+        self.yql_create_group(self.tenant_admin_driver_config)
+
+    def test_query_create_user_by_domain_admin(self):
+        self.query_create_user(self.domain_admin_driver_config)
+
+    def test_query_create_user_by_tenant_admin(self):
+        self.query_create_user(self.tenant_admin_driver_config)
+
+    def test_query_create_group_by_domain_admin(self):
+        self.query_create_group(self.domain_admin_driver_config)
+
+    def test_query_create_group_by_tenant_admin(self):
+        self.query_create_group(self.tenant_admin_driver_config)
+
+
+class TestCreateSubjectsWithDomainLoginOnly(BaseCreateSubjects):
+    DomainLoginOnly = True
+
+
+class TestCreateSubjectsWithoutDomainLoginOnly(BaseCreateSubjects):
+    DomainLoginOnly = False
```

Comment:
Also pytest has plenty of ways to conveniently [parametrize test functions and classes](https://docs.pytest.org/en/stable/how-to/parametrize.html#parametrize).